lib/Arch/AArch32/Decode.cpp

```diff
@@ -2253,8 +2302,13 @@ static bool TryLogicalArithmeticRRRI(Instruction &inst, uint32_t bits) {
     AddImmOp(inst, ~0u);
   }
 
-  AddShiftRegImmOperand(inst, enc.rm, enc.type, enc.imm5, enc.s);
-  return EvalPCDest(inst, enc.s, enc.rd, kLogArithEvaluators[enc.opc >> 1u],
+  if (enc.type) {
+	  AddShiftRegImmOperand(inst, enc.rm, enc.type, enc.imm5, enc.s);
```

Comment:
Run code formatter to fix indentation; `./scripts/format-files` I believe. Make sure to copy the `format-files` from anvill first; I think it's script is more up-to-date.